lib/lrama/report.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
-require_relative 'report/duration'
+require_relative 'report/conflicts'
```

Comment:
What is the reason why keeping `Report` module even so introducing `Reporter`?

Reply:
Same as: https://github.com/ruby/lrama/pull/534/files#r1952220404

Reply:
fixed in: https://github.com/ruby/lrama/pull/534/commits/62c5593fa2e83b2420dfd88259191b622f214ffa

Reply:
refixed: https://github.com/ruby/lrama/pull/534/commits/4026c1dc1451f9d7ac7bf03838259d8e7edd4d0e